examples/switch.rb

```diff
@@ -20,9 +20,10 @@
 }
 
 # Example: No action is executed.
-# In a resource that has only one action and no action is specified in the block
-# Chef executes this one. To prevent Chef from removing a switch as the standard action we created
-# the none action
+# In a resource that only has a remove action and other actions without great relevance to be considered
+# a standard action and no action is specified in the block Chef executes this one.
```

Comment:
I can't understand what you've said there, but I guess it should be:
> In a resource that only has destructive and non intuitive actions, Chef executes the none action to avoid mistakes.